client/src/Client.js

```diff
@@ -0,0 +1,154 @@
+/* eslint-disable no-console */
```

Comment:
Why do these these lints are disabled?

Reply:
This was copied over from some starting code files from the book.
I'll remove them.